src/components/loader.js

```diff
@@ -24,11 +21,11 @@ module.exports.Component = registerComponent('loader', {
     var url = parseUrl(data.src);
     var mtlUrl = parseUrl(data.mtl);
     var format = data.format;
+
     if (model) { el.removeObject3D('mesh'); }
-    if (!url) {
-      warn('Model URL not provided');
-      return;
-    }
+
+    if (!url) { return console.warn('Model `src` was not specified'); }
```

Comment:
We should use ```debug.js``` and enable debug logs by default in a different PR. It will be annoying to chase down the console.log calls later on.